sale_order_report_hide_discounts_by_partner/README.rst

```diff
@@ -0,0 +1,95 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+	:target: http://www.gnu.org/licenses/agpl
+	:alt: License: AGPL-3
+
+===========================================
+Sale Order Report Hide Discounts by Partner
+===========================================
+
+This module allows you to show the discounts on sale reports if based on the partner.
+
+In order not to show the discounts in the sale reports, on the one hand hide the discount columns and 
+on the other hand recalculate the unit price including the discount.
+
+
+Installation
+============
+
+To install this module, you need to:
+
+#. Only install
+
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Go to the contacts module 
+#. Create or Edit a parent contact
+#. Go to Sale & Purchase tab
+#. Edit the 'Show discounts' field
```

Comment:
Te name of the field is "Show Discounts in Sales", right?

Reply:
Correct, now I change it